src/Get-Help/User-Guide/Installation/index.md

```diff
@@ -67,6 +67,105 @@ The ZIP can be installed on all Linux distros. It includes the desktop
 application, the server and the command line tool. To install, simply
 extract the contents to any destination you want.
 
+### Docker
+
+The Docker distribution is not available as a download on the
+website. It comes in the form of a Docker image that you can obtain
+via the Docker command line interface:
+
+~~~sh
```

Comment:
I thought the images are pulled automatically if you type `docker run image`, so I wouldn't even mention the docker pull commands.
This of course depends on who your audience is. I envision someone like Mischa that doesn't care that much about the technical details and just want the pipeline up and running

Reply:
Ah yes you're right.